docs_src/openapi_overview/tutorial001.py

```diff
@@ -0,0 +1,95 @@
+from fastapi import FastAPI, Path, Query
+from pydantic import BaseModel, Field
+
+app = FastAPI(
+    title="The title of the API",
+    description="""
+possibly lengthy description of the api including
+
+newlines and `formating`
+    """,
+    version="Any string but please use semver (1.0.0-beta)",
+)
+
+
+class CustomError(BaseModel):
+    cause: str
+
+
+class Item(BaseModel):
+    id: str = Field(
+        title="Short title of the field",
+        description="Looooooooonger description what the filed is used for etc.",
```

Comment:
I would add some line break to show this is possible. I did not manage yet to include a table but I am sure this can be done. If you know how to do it?? Also, it seems like using docstrings instead of strings is not possible here @tiangolo ?
```suggestion
        description="Looooooooonger description what the filed is used for etc.\n\nThis is a line break.",
```